.eslintrc

```diff
@@ -11,6 +11,11 @@
     "jest": true,
     "jasmine": true
   },
+  "globals": {
+    "fixture": true,
+    "$": true,
```

Comment:
we are aiming to not having jQuery as a global, at least not in our modules

Reply:
OK, this is wrong then.